src/styles/blocks/button.scss

```diff
@@ -0,0 +1,26 @@
+.button {
+  cursor: pointer;
+  box-sizing: border-box;
```

Comment:
Move to the * selector and remove from all other places

Reply:
based on everything, I think that my tests did not pass, due to the fact that I just did revert and merge, and as you can see, for some reason I have more tests than others (28 tests for me, 14 for others), I think for some reason I still have old tests.
I think it’s okay, I’m just hoping for a mentor check)
Thank you.

Reply:
and i have now border-box only in 1 place, so i dont wanna do  * { }